rholang/README.md

```diff
@@ -48,7 +48,6 @@ In general:
   * 0-arity send and receive is currently broken.
   * Matching is incomplete. We won't currently destructure receives or matches. We will match expressions.
   * We don't pre-evaluate match cases. So matching 7 + 8 as a pattern currently doesn't work. Instead, you must match against 15.
-  * There is work in progress to support stdout and stderr, but it hasn't landed yet.
   * There is also work to support native functions. It hasn't landed yet.
 ### The good
-Several working examples have been included in the examples/0.2 directory, and the tests in tests/mercury-tut also work. If you run into something that doesn't work, check the bugtracker to see if it's a known issue, and if not, feel free to file a bug. We want rholang to be a useful programming environment.
+Several working examples have been included in the examples directory, and the tests in the [Rholang tutorial](https://github.com/rchain/rchain/blob/master/docs/rholang/rholangtut-0.2.md) also work. If you run into something that doesn't work, check the bugtracker to see if it's a known issue, and if not, feel free to [file a bug](https://rchain.atlassian.net/secure/CreateIssueDetails!init.jspa?pid=10105&issuetype=10103&versions=10012&components=10004&assignee=medha&summary=issue+created%20via+link). We want Rholang to be a useful programming environment.
```

Comment:
The tut has been updated to 0.3.  I'd also call them examples rather than tests.